profile/README.md

```diff
@@ -1,49 +1,48 @@
-
-## Vibinex Code Review
-This is our flagship product ([vibinex.com](https://vibinex.com)). It modifies the UX for code-hosting websites like GitHub and Bitbucket to make it easier to perform high-quality code reviews.
-There are three key repositories of this product:
-1. [User-side data processing unit (Rust)](https://github.com/Alokit-Innovations/vibi-dpu)
-2. [Server for backend and website (NextJS)](https://github.com/Alokit-Innovations/vibinex-server)
-3. [Browser Extension](https://github.com/Alokit-Innovations/chrome-extension)
+**Vibinex is an open-source privacy-first code review automation tool.**
+
+- Get your pull request merged quickly
+- Understand the changes in a pull request faster and better
+- Reduce the average time to first review by 50%
+
+
+[<kbd> <br> Get Started <br> </kbd>](https://vibinex.com/docs/setup/providerLogin)
+
+<sub>[Website](https://vibinex.com/) &nbsp; • &nbsp; [Documentation](https://vibinex.com/docs) &nbsp; • &nbsp; [GitHub](https://github.com/vibinex/vibinex-server)</sub>
+
+**Vibinex has three key repositories:**
+
+1. [vibi-dpu - User-side data processing unit (Rust)](https://github.com/vibinex/vibi-dpu)
+2. [vibinex-server - Server for backend and website (NextJS)](https://github.com/vibinex/vibinex-server)
+3. [chrome-extension - Browser Extension](https://github.com/vibinex/chrome-extension)
+
+<br/>
+<br/>
+<br/>
 
 <picture>
-  <source media="(prefers-color-scheme: dark)" srcset="https://github.com/Alokit-Innovations/.github/assets/7858932/1246e2b8-9ba9-4e27-af30-b159b9c8e9bb">
-  <source media="(prefers-color-scheme: light)" srcset="https://github.com/Alokit-Innovations/.github/assets/7858932/1530e2d0-b118-484f-84a1-ef26ab305326">
-  <img alt="Vibinex core repo" src="https://github.com/Alokit-Innovations/.github/assets/7858932/1530e2d0-b118-484f-84a1-ef26ab305326">
+  <source media="(prefers-color-scheme: dark)" srcset="https://github.com/vibinex/.github/assets/65667944/7bf2ca92-2313-4411-bf76-cc5baf127edd">
+  <source media="(prefers-color-scheme: light)" srcset="https://github.com/vibinex/.github/assets/65667944/fe874d58-1ed9-401d-9208-9fc755e18b22">
+  <img alt="Vibinex core repo" src="https://github.com/vibinex/.github/assets/65667944/fe874d58-1ed9-401d-9208-9fc755e18b22">
 </picture>
 
-### Privacy-first
-We take privacy extremely seriously. The purpose of the Rust service is to keep the user's data (code, events, config, server logs, etc.) in the user's infrastructure.
-Only non-reproducible encrypted/hashed metadata is shared with Vibinex's servers. 
+<br/>
+<br/>
+<br/>
+
+**Architecture:**
 
-This project follows a dual-backend architecture to ensure privacy while serving data-enabled features. Here is the architectural diagram (over-simplified for high-level understanding):
+Vibinex follows a dual-backend architecture to ensure privacy while serving data-enabled features. Here is the architectural diagram (over-simplified for high-level understanding):
```

Comment:
AJ: included architecture section, making it more functional than feature specific. User will understand the functional parts of Vibinex.